google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -1113,8 +1117,13 @@ public void bulkMutateRows(BulkMutation mutation) {
 
   /**
    * Reads rows for given tableId and filter criteria in a batch. If the row does not exist, the
-   * value will be null. This operation should be called with in a single thread. The returned
-   * Batcher instance is not threadsafe, it can only be used from single thread.
+   * value will be null. The returned Batcher instance is not threadsafe, it can only be used from a
+   * single thread.
+   *
+   * <p>Performance notice: The ReadRows protocol requires that rows are sent in ascending key
+   * order, which means that the keys are processed sequentially on the server-side, so batching
+   * allows improving throughput but not latency. Low latency might be achieved if send smaller
+   * requests concurrently.
```

Comment:
Lower latencies can be achieved by sending smaller requests concurrently.

Reply:
Agree, sounds much better.